packages/react-dom/src/events/getEventCode.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @see https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code#Code_values
+ * @param {object} nativeEvent
+ * @returns {string} Standardized keyboard code value if defined natively
+ */
+function getEventCode(nativeEvent: KeyboardEvent): string {
+  if (__DEV__ && nativeEvent.code === undefined) {
+    throw new ReferenceError(
+      'KeyboardEvent.code does not exist in this environment',
```

Comment:
The message is not particularly clear. If we go for it, we need to make sure that it explains what's going on. What about:

> KeyboardEvent.code is not supported in the browser you’re currently using. You should either add a polyfill or use KeyboardEvent.key instead.